Maple2.Model/Metadata/FieldEntity/FieldEntity.cs

```diff
@@ -10,7 +10,8 @@ public enum FieldEntityType : byte {
     BoxCollider, // for cube tiles (IsWhiteBox = false) & arbitrarily sized white boxes
     MeshCollider,
     Fluid,
-    Cell // for cells culled from the grid & demoted to AABB tree
+    Cell, // for cells culled from the grid & demoted to AABB tree
+    SalableTile,
```

Comment:
You can move SalableTile above Cell if you want. Might make sense to keep Cell outside of the other entity types because it's more for internal use